src/sentry/eventstore/base.py

```diff
@@ -169,6 +172,12 @@ def get_latest_event_id(self, event, filter):
         """
         raise NotImplementedError
 
+    def from_data(self, project_id, event_id, group_id, data):
```

Comment:
What about just `create_event` ?

Reply:
Sure. I was trying to come up with something that didn't imply saving.